dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -46,6 +46,13 @@ service AppCallback {
   rpc OnBindingEvent(BindingEventRequest) returns (BindingEventResponse) {}
 }
 
+// AppCallbackHealthCheck V1 is an optional extension to AppCallback V1 to implement
+// the HealthCheck method.
+service AppCallbackHealthCheck {
```

Comment:
This is implemented as a separate service so it's optional for clients and preserves backwards-compatibility